lms/djangoapps/teams/models.py

```diff
@@ -48,9 +49,11 @@ def create(cls, name, course_id, description, topic_id=None, country=None, langu
         """
 
         team_id = generate_unique_readable_id(name, cls.objects.all(), 'team_id')
+        discussion_id = 'team_id:' + team_id
```

Comment:
The goal is to create unique ids for the comments service that are unlikely to conflict with real discussion topics. team_ids are unique, and the prefix helps prevent unwanted conflicts. For example, a course could have a team with id "solar-power" and a discussion topic "solar-power"